lib/index.js

```diff
@@ -33,18 +33,21 @@ exports.plugin = {
     pkg: require('../package.json'),
     once: true,
     requirements: {
-        hapi: '>=19.0.0'
+        // NOTE v21 beta is included only prior to the release of v21, and will be removed.
+        hapi: '>=20.0.0 || 21.0.0-beta.0'
```

Comment:
should we create an issue in inert to make sure we don't forget about it or are you noting it somewhere already @devinivy?

Reply:
Doesn't `>=20.0.0` already cover `21.0.0-beta.0`?

Reply:
One might think so, but they do not match due to an exception made for pre-releases. Here's a similar example from the test suite: https://github.com/hapijs/somever/blob/5080bd4e3aa4f68177d583fc7699996c566b09ed/test/index.js#L146.  And here's a related issue: https://github.com/hapijs/somever/issues/18

Reply:
Ah, right you are – I have been bitten by this before 🤦

I would make it a range through, eg. `^21.0.0-beta.0` to ensure it also works with other betas.